libs/withdraws/src/lib/withdraw-form.tsx

```diff
@@ -190,15 +191,12 @@ export const WithdrawForm = ({
 interface UseButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   children: ReactNode;
 }
-
 const UseButton = ({ children, ...rest }: UseButtonProps) => {
   return (
-    <button
-      {...rest}
-      type="button"
-      className="ml-auto text-sm absolute top-0 right-0 underline"
-    >
-      {children}
-    </button>
+    <span className="ml-auto text-sm absolute top-0 right-0">
```

Comment:
Chore (blocking): another case of specifying a `relative` on a parent to ensure `absolute` is consistent across browsers

Reply:
As above.